gulpfile.js

```diff
@@ -0,0 +1,22 @@
+var gulp     = require('gulp'),
+    mocha    = require('gulp-mocha'),
+    babel    = require('babel'),
+    plumber  = require('gulp-plumber'),
+    istanbul = require('gulp-istanbul');
```

Comment:
if you really want to go that way I can cope, but I think how we've been doing it at work is nice ie

node packages

company packages

project includes

etc